themes/_base.sass

```diff
@@ -0,0 +1,11 @@
+@mixin component($component)
+  .ais-#{$component}
+    @content
+
+@mixin modifier($modifier)
+  &--#{$modifier}
+    @content
+
+@mixin element($element)
+  &__#{$element}
+    @content
```

Comment:
How would you use that?

Hmm, the goal of BEM is to not have to do any nesting. Sass can let you easily do a lot of nesting, but we should actually never nest more than 2 or 3 levels deep.